components/migratecdp/PayAndMigrate.js

```diff
@@ -352,13 +359,14 @@ const PayAndMigrate = ({
               ) : null}
             </Table.tbody>
           </Table>
-          {aboveOneSeventy ? (
+          {aboveOneSeventy && enoughMkrInMarket ? (
             <TOSCheck {...{ hasReadTOS, setHasReadTOS }} />
           ) : (
             <ErrorBlock>
-              You cannot use this feature because your CDP would end up with a
-              collateralization ratio of less than {CDP_MIN_RATIO}%. Please use
-              ‘Pay with MKR’ or repay some of your CDP debt before continuing.
+              {!aboveOneSeventy ? `You cannot use this feature because your CDP would end up with a
+              collateralization ratio of less than ${CDP_MIN_RATIO}%. Please use
+              ‘Pay with MKR’ or repay some of your CDP debt before continuing.`
+              : `There is not enough liquidity in the market at this time to purchase ${selectedCDP.govFeeMKR} MKR with a maximum of 5% slippage.`}
```

Comment:
I don't know if we should assume that all users understand what's happening under the hood when they're paying with debt, so maybe being a little more specific would make more sense? E.g. `There is not enough MKR liquidity in Oasis[...]` or something

Reply:
I think this is ok for now. It's more important we get something out. Can I just check that we are also not going to enable the continue button if they see this error?

Reply:
RIght, we don't show the continue button if they see this error.  I'll go ahead and merge this then